routes/feed.js

```diff
@@ -1,22 +1,29 @@
 var express = require('express');
-var router = express.Router();
-
+var moment = require('moment');
 var itemGenerator = require('../data/itemGenerator');
+var env = require('../env');
 
-var moment = require('moment');
+var router = express.Router();
 
-moment.fn.roundNextMin = function (min) {
-  if (60 % min != 0) throw "60 must be divisible by input: " + min; 
-  var intervals = Math.floor(this.minutes() / min);
+/**
+ * Round a date to the next interval (e.g. to the next hour)
+ *
+ * @param {number} intervalMinutes
+ * @param {import('moment').Moment} momentDate
+ */
+function roundNextMin(intervalMinutes, momentDate) {
+  const newMomentDate = moment(momentDate);
+  if (60 % intervalMinutes !== 0) throw new Error('60 must be divisible by `intervalMinutes`. Value: "' + intervalMinutes + '"');
+  var intervals = Math.floor(newMomentDate.minutes() / intervalMinutes);
   intervals++;
-  if(intervals == 60/min) {
-    this.add('hours', 1);
+  if(intervals == 60/intervalMinutes) {
+    newMomentDate.add(1, 'hours');
     intervals = 0;
   }
-  this.minutes(intervals * min);
-  this.seconds(0);
-  this.milliseconds(0);
-  return this;
+  newMomentDate.minutes(intervals * intervalMinutes);
+  newMomentDate.seconds(0);
+  newMomentDate.milliseconds(0);
+  return newMomentDate;
 }
```

Comment:
Sorry, this is a refactor for no particular reason other than the tools I use for error checking don't work when prototypes are modified like this. Prototype modification is generally considered bad practice, so I made this quick change